test/SILGen/hop_to_executor_async_prop.swift

```diff
@@ -648,22 +648,22 @@ struct StateObject<ObjectType> {
     init(wrappedValue: ObjectType) {}
 }
 
-final private actor Coordinactor {
+final private actor Coordinator {
     var someValue: Int?
 }
 
 struct Blah {
-    @StateObject private var coordinator = Coordinactor()
+    @StateObject private var coordinator = Coordinator()
```

Comment:
I can't tell if this is an intentional cross or a typo.

Reply:
@kavon is this an intentional spelling you'd like to keep in the test?